examples/with-react-relay-network-modern/pages/_app.js

```diff
@@ -27,17 +37,19 @@ export default class App extends NextApp {
 
     return {
       variables,
+      store,
```

Comment:
Did you test that this works properly? only values that can be valid JSON are allowed to be returned in props, `store` here is a class

Reply:
Yeah it seemed to work for me, needed a way to share the same store for `initEnvironment` and `createEnvironment` methods for SSR. We're not using the store prop at the client side.

Reply:
@Timer Can you confirm that this will work properly :pray:

Reply:
in my opinion you should create the store every time but send the state and hydrate the new store with the props returned in data fetching.

Reply:
Yeah that seems a better option, if we're able to init the store using the `relayData` like we do for `queryRecords` in the with-relay-modern example

I was just wondering if `QueryRenderer` should provide the data before rendering, why isn't it fetching from the network in this case

Reply:
Does this seem better: https://github.com/rishabhsaxena/next.js/commit/96b49f2727d20bf5c06a97107d11644f1baab848 ?

Have used records to initate store as:
```
const records = environment
  .getStore()
  .getSource()
  .toJSON()
```

Did not need to use `react-relay-network-modern-ssr` for this though

Reply:
I'm not sure, I don't know how Relay works, but once you update the example I can take a look into the code, test that it works and review it.

Reply:
@lfades  Ok sure, thanks. Have created another PR: https://github.com/zeit/next.js/pull/11343